ang/civicase/ActivityCard--Long.html

```diff
@@ -1,6 +1,6 @@
 <div
   class="civicase__activity-card civicase__activity-card--long"
-  ng-click="feedViewFn ? feedViewFn(activity.id, $event) : viewInPopup($event, activity)"
+  ng-click="viewInPopup($event, activity)"
```

Comment:
For every Activity Card, we should write `viewActivityDetails($event)`.
Inside the Activity card.
```
$scope.viewActivityDetails = function () {
  $scope.$emit();

  if ($scope.viewActivityInPopup) {
     $scope.viewInPopup();
  }
}
```